src/pages/jobs.astro

```diff
@@ -0,0 +1,111 @@
+---
+import { Image } from 'astro:assets';
+import MainLayout from '../layouts/MainLayout.astro';
+import MainSection from "../components/ui/blocks/MainSection.astro";
+import FreelancerCard from '../components/FreelancerCard.astro';
+import freelancers from '../data_files/freelancers.json';
+// Import PrimaryCTA component
+import PrimaryCTA from "@components/ui/buttons/PrimaryCTA.astro";
+
+const pageTitle = "Available Spryker Developers";
+
+// Type definitions for the Freelancer object
+type Freelancer = {
```

Comment:
what about moving it to https://github.com/spryker-community/CommerceQuest-website/blob/main/src/types.ts ? because i wouldnt just look for a freelancer-"type" in a jobs-page file

Reply:
Done